rand_distr/src/poisson.rs

```diff
@@ -67,6 +67,7 @@ where Standard: Distribution<N>
 impl<N: Float> Distribution<N> for Poisson<N>
 where Standard: Distribution<N>
 {
+    #[inline]
```

Comment:
This is a large function — I don't think `#[inline]` makes any sense here.

Reply:
This is not `#[inline(always)]`, so the compiler is still free to make that choice. It might make sense to inline it into the `u64` sampling, so that some bound checks can be eliminated.